src/GalleryApp.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Header from './Components/Header';
+import Footer from './Components/Footer';
+
+class GalleryApp extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: null,
+      isLoaded: false,
+      items: []
+    }
+  }
+
+  componentDidMount() {
+    fetch("https://jsonplaceholder.typicode.com/photos")
+    .then(res => res.json())
+    .then(
+      (result) => {
+        this.setState({
+          isLoaded: true,
+          items: result
+        });
+      },
+      (error) => {
+        this.setState({
+          isLoaded: true,
+          error
+        });
+      }
+    )
+  }
+
+  render() {
+    const { error, isLoaded, items } = this.state;
+
+    if (error) {
+      return <div>Error: {error.message}</div>;
+    } else if (!isLoaded) {
+      return <div className='loader'></div>
+    } else {
+      return (
+        <div className="gallery-app">
+
+          <Header headingProp={'Gallery list'} />
+
+          <div className="gallery-app-body">
+            <div className="listed-images">
+              <ul>
+                {items.map(item => (
```

Comment:
Should deconstruct your items here so that you don't need to keep having item. later on

Reply:
Added destructured props 'item' from 'items' in this.state()
eg; const {albumId, id, title, thumbnailUrl, url} = item;
Added inside the map() function, although i had to refactor and add a second return within the map function to make it work, I need to understand this or a better way to do it.